app/css/modals/become-mentor.css

```diff
@@ -81,21 +81,12 @@
             @apply grid gap-20;
             grid-template-columns: repeat(4, auto);
 
-            & .track {
-                @apply bg-backgroundColorB shadow-lg rounded-8;
-                @apply py-24;
-                @apply flex flex-col items-center;
-                @apply relative;
-
-                cursor: pointer;
-                transition: background 0.2s ease-out, box-shadow 0.2s ease-out;
-
-                &:hover {
-                    @apply shadow-lgZ1;
-                    @apply bg-backgroundColorA;
+            & .track-checkbox {
```

Comment:
@iHiD I changed the CSS and HTML a bit in order to make the checkbox and label accessible via keyboard shortcuts. I've added a focus-visible style to the label whenever the checkbox is focused.